scripts/files/file_tiff.py

```diff
@@ -307,25 +302,6 @@ def check_color_interpretation(self, gdalinfo: GdalInfo) -> None:
                 custom_fields={"missing": f"{', '.join(missing_bands)}"},
             )
 
-    def check_tile_and_rename(self, gdalinfo: GdalInfo) -> None:
```

Comment:
This functionality is now handled in https://github.com/linz/argo-tasks/pull/508

Reply:
Is there any reason to remove this? could we leave this  for if there is one tile in one tile out?

Reply:
In the case of running a workflow with Argo, for the case of one tile in and one tile out `{"output": "BX24_500_031020", "inputs": "BX24_500_031020.tif"}`, the `output` has been generated by `argo-tasks` so we assume it's correct? Or do we have a case without validation from `argo-tasks`? 
In a case of local usage (mostly dev) that could be useful.

Reply:
I was mostly thinking about making it easier for local dev/testing